src/glade.py

```diff
@@ -104,7 +104,8 @@ def gen_rep(alpha):
     for i in range(length): # shorter alpha1 prioritized
         alpha_1 = alpha[:i]
         # alpha_2 != e
-        for j in range(i+1, length+1): # longer alpha2 prioritized
+        for k in range(i+1, length+1): # longer alpha2 prioritized
+            j = length - (k - (i+1))
```

Comment:
Hey, any chance you can tie it back to the paper (Page, Section,  subsection + a line of comment would be ideal).

Reply:
Page: 5, section: 4.2. 
Quote: "we first prioritize shorter a_1, since a_1 is not further generalized. Second, we prioritize longer a_2"
The code implementation before my suggested modification prioritized smaller a_2. That can be seen from  the "j" index", whereas j should be made to start from "length+1" and decrement until "i+1".

Reply:
What I meant is in the checkin, add as a comment in glade.py where you modify it.